pkg/cmd/scan.go

```
@@ -154,6 +157,15 @@ func NewScanCmd() *cobra.Command {
 		"Includes cloud provider service-linked roles (disabled by default)",
 	)
 
+	configDir, _ := homedir.Dir()
```

Are you sure it's safe to ignore the error here?

---

Yes the test can demonstrate multiple situations, and if configDir is "" et err != nil, this variable will be handle after that to call tmpDir()

---

Not sure to understand, I don't see any fallback to tmpDir() in the diff

---

pkg/terraform/provider_installer.go:78

---

I think what @sundowndev meant is that for the default variable just below `configDir` if `homedir.Dir()` returns an error, we won't have any default displayed inside the `--help` message. Is that it ?

---

Could you put the fallback here instead ? I think it's confusing that it's not close to the definition